cartridges/int_turnto_core_v5/cartridge/scripts/jobsteps/ExportCatalog.js

```diff
@@ -175,7 +175,7 @@ function process(product) {
         var categoryPathJSON = null;
         var currentCategory =
             product.getPrimaryCategory() ||
-            (product.isVariant() && product.masterProduct.getPrimaryCategory()) ||
+            product.isVariant() ? product.masterProduct.getPrimaryCategory() : null ||
```

Comment:
@brianshenPixleeTurnto I think we need to check masterProduct exists
product.getPrimaryCategory() ||
            (product.isVariant() && product.masterProduct) ? product.masterProduct.getPrimaryCategory() : null